openapi_tester/schema_tester.py

```diff
@@ -185,20 +185,24 @@ def _validate_key_casing(
             tester(key)
 
     @staticmethod
-    def _validate_enum(schema_section: dict, data: str) -> Union[Optional[str], bool]:
-        if "enum" not in schema_section:
+    def _validate_enum(schema_section: dict, data: Any) -> Union[Optional[str], bool]:
+        enum = schema_section.get("enum")
+
+        if enum is None:
```

Comment:
if not enum rather than using is.

Reply:
''return not enum or data in enum''

Also, you can put as default value to get an empty list.

Reply:
Updated 👍 453a53e